SD_Card_Driver/Core/Inc/LOS_D_sd.hpp

```diff
@@ -0,0 +1,29 @@
+#ifndef LOS_D_SD_H_
+#define LOS_D_SD_H_
+
+#include "fatfs.h"
+#include "string.h"
+#include "LOS_D_storage_device.hpp"
+
+class SDCard : public StorageDevice {
+	private:
+		FATFS fs;
+		const char* curOpenFilePath;

```

Comment:
ok I get to be picky & really annoying rn.

Idk what the rest of the teams thoughts are on this but typically variable names are snake_case.....

Reply:
yes, I believe the convention would snake_case for variable names then camelCase for functions...